src/setts/setts.service.ts

```diff
@@ -82,27 +86,46 @@ export class SettsService {
       sources: [],
     };
 
-    const [settSnapshots, settData] = await Promise.all([
-      this.getSettSnapshots(settName, SAMPLE_DAYS),
-      getSett(chain.graphUrl, settDefinition.settToken),
-    ]);
+    const record = await query({
+      TableName: SETT_SNAPSHOTS_DATA,
+      KeyConditionExpression: '#address = :address',
+      ExpressionAttributeValues: {
+        ':address': { S: settDefinition.settToken },
+      },
+      ExpressionAttributeNames: {
+        '#address': 'address',
+      },
+    });
 
-    // set to current balance, fallback to snapshot
+    // Set to current balance, fallback to snapshot
     let balance = 0;
-    if (settData.sett) {
-      const currentSett = settData.sett;
-      balance = currentSett.balance;
-      sett.ppfs = currentSett.balance / currentSett.totalSupply;
+    let settSnapshots: SettSnapshot[] = [];
```

Comment:
the sett snapshots are also used for calculating the sampled apy based on historic ppfs delta, so we want to replace the logic that is poulating `settData` which is the "live lookup" vs. replacing the historic data.

Reply:
to try to clear it up basically:

sett snapshots = the historic 30 min sampled values for estimating native apy,
sett data = the current state of the sett

so these cached ones need to replace the second as the apy calculation are assuming a set of data at 30 min polling intervals

Reply:
Discussed offline 👍